changelogs/current.yaml

```diff
@@ -112,6 +112,12 @@ new_features:
   change: |
     added :ref:`custom_sink <envoy_v3_api_field_config.tap.v3.OutputSink.custom_sink>` type to enable writing tap data
     out to a custom sink extension.
+- area: tcp
+  change: |
+    added the support to detect and send TCP RST for raw buffer socket based connection, and this is disabled by default.
+    This literally is supported for linux. Windows also works but some behaviors are different.
```

Comment:
I'm not completely understand this line. Did you mean to say: "This feature is fully supported on Linux."? For Windows support we either need to fully disable it, or document what is different.